website/content/en/docs/setup/going-to-prod/arch/aggregator.md

```diff
@@ -0,0 +1,114 @@
+
+{{< info >}}
+See the [high availability document](/docs/setup/going-to-prod/high-availability/) for more detail.
+{{< /info >}}
+
+### Hardening
+
+- [Secure Vector’s data](/docs/setup/going-to-prod/arch/hardening/#securing-the-data).
+- [Secure the Vector process](/docs/setup/going-to-prod/arch/hardening/#securing-the-vector-process).
+- [Secure the host](/docs/setup/going-to-prod/arch/hardening/#securing-the-host).
+- [Secure the network](/docs/setup/going-to-prod/arch/hardening/#securing-the-network).
+
+{{< info >}}
+See the [hardening recommendations](/docs/setup/going-to-prod/arch/hardening/) for more detail.
+{{< /info >}}
+
+### Sizing, Scaling, & Capacity Planning
+
+- [Front your aggregators with a load balancer](/docs/setup/going-to-prod/sizing/).
+- Provision at least [4 vCPUs](/docs/setup/going-to-prod/sizing/#cpus) and [8 GiB of memory](/docs/setup/going-to-prod/sizing/#memory) per instance.
+- Enable [autoscaling](/docs/setup/going-to-prod/sizing/#autoscaling) with a target of 85% CPU utilization.
+
+{{< info >}}
+See the [sizing, scaling, and capacity planning document](/docs/setup/going-to-prod/sizing/) for more detail.
+{{< /info >}}
+
+### Rolling Out
+
+- Roll out [one network partition and one system at a time](/docs/setup/going-to-prod/rollout/#incremental-adoption).
+- Following the roll-out [strategy](/docs/setup/going-to-prod/rollout/#rollout-strategy) and [plan](/docs/setup/going-to-prod/rollout/#rollout-plan).
+
+{{< info >}}
+See the [rolling out document](/docs/setup/going-to-prod/rollout/) for more detail.
+{{< /info >}}
+
+## Advanced
+
+### Pub-Sub Systems
+
+{{< warning >}}
+We do not recommend provisioning a new pub-sub service for the sole purpose of Vector. Vector can deploy in a [highly available manner](/docs/setup/going-to-prod/high-availability/) that minimizes the need for such systems.
+{{< /warning >}}
+
+The aggregator architecture can deploy as a consumer to a pub-sub service, like Kafka:
+
+![Aggregator](/img/going-to-prod/pub-sub.png)
+
+#### Partitioning
+
+Partitioning, or “topics” in Kafka terminology, refers to separating data in your pub-sub systems. We strongly recommend partitioning along data origin lines, such as the service or host that generated the data.
+
+![Aggregator](/img/going-to-prod/partitioning.png)
+
+#### Recommendations
+
+- If you’re using your pub-sub system purely for durability, consider Vector’s external buffers, which buffers data on your pub-sub system while transparently handling data producing and consuming.
+- Use memory buffers with `buffers.when_full` set to `block`. This will ensure back pressure flows upstream to your pub-sub system, where durable buffering should occur.
+-  Enable [end-to-end acknowledgements](/docs/about/under-the-hood/architecture/end-to-end-acknowledgements/) for your Vector pub-sub source (i.e., the `kafka` source) to ensure data is persisted downstream before removing the data from your pub-sub systems.
```

Comment:
Technically it is still on the source right now, but agreed, this will need to be updated for the next release where it will be moved to the sink. Given this will be published first, we should follow up to update this.